### PR TITLE
contracts: update VRFExternalSubOwnerExample

### PR DESCRIPTION
**Pull request by Nic0S**
- Allow specifying a subscription ID and other VRF params in
   requestRandomWords()

 - Add transferOwnership function

 - Update tests and scripts for new interface

 - Also update the "kovan" script to take chain ID as an env var
   and rename to "testnet"

### CODE REVIEW

**Review by makramkd — DISMISSED** [review]
:shipit:

**Comment by makramkd** [comment]
One additional operation that I found useful in my testing is to check the subscription balance for a particular subscriber:

```golang
// ... rest of switch ...
case "sub-balance":
        consumerBalanceCmd := flag.NewFlagSet("sub-balance", flag.ExitOnError)
        coordinatorAddress := consumerBalanceCmd.String("coordinator-address", "", "coordinator address")
	subID := consumerBalanceCmd.Uint64("sub-id", 0, "subscription id")
	panicErr(consumerBalanceCmd.Parse(os.Args[2:]))
	coordinator, err := vrf_coordinator_v2.NewVRFCoordinatorV2(common.HexToAddress(*coordinatorAddress), ec)
	panicErr(err)
	resp, err := coordinator.GetSubscription(nil, *subID)
	panicErr(err)
	fmt.Println("sub id", *subID, "balance:", resp.Balance)
```

**Comment by Nic0S** [comment]
Nice that's a good idea, went ahead and added it to scripts

**Review by makramkd — APPROVED** [review]
🚢

File: core/services/vrf/integration_v2_test.go [hunk 1]
```diff
@@ -487,7 +487,7 @@ func TestExternalOwnerConsumerExample(t *testing.T) {
 	})
 	require.NoError(t, err)
 	backend.Commit()
```
Comment (connorwstein) [inline core/services/vrf/integration_v2_test.go:489]: could add a quick ownership transfer test here

Comment (Nic0S) [reply core/services/vrf/integration_v2_test.go:490]: done!

**Pull request merged**